monai/auto3dseg/utils.py

```diff
@@ -270,10 +270,10 @@ def verify_report_format(report: dict, report_format: dict):
             else:
                 return False
 
-        return True
+    return True
```

Comment:
I think this could have been a bug since the loop would always finish in the first iteration.